services/sse/pkg/config/config.go

```diff
@@ -14,7 +15,8 @@ type Config struct {
 	Debug   Debug    `mask:"struct" yaml:"debug"`
 	Tracing *Tracing `yaml:"tracing"`
 
-	Service Service `yaml:"-"`
+	Service           Service       `yaml:"-"`
+	KeepAliveInterval time.Duration `yaml:"keepalive_interval" env:"SSE_KEEPALIVE_INTERVAL" desc:"To prevent intermediate proxies from closing the SSE connection send periodic SSE comments." introductionVersion:"6.7"`
```

Comment:
```suggestion
	KeepAliveInterval time.Duration `yaml:"keepalive_interval" env:"SSE_KEEPALIVE_INTERVAL" desc:"Send periodic SSE 'comments' to prevent intermediate proxies from closing the SSE connection." introductionVersion:"7.0"`
```
Expecting that this goes into 7.0
Q: have not found it, just for my knowledge, is `time.Duration` a duration?